kleeanalysis/kleedir/kleedir.py

```diff
@@ -64,10 +64,16 @@ def __init__(self, path: "Path to a KLEE working directory."):
             self.tests.append(Test(test_file_path))
 
 
-        with open(os.path.join(path, "messages.txt")) as file:
-            self.messages = file.readlines()
-        with open(os.path.join(path, "warnings.txt")) as file:
-            self.warnings = file.readlines()
+        try:
```

Comment:
This change seems sensible. If you split this into a separate commit I'd happily accept it.